rpm/containers-common.spec

```diff
@@ -85,19 +85,21 @@ Summary: Extra dependencies for Podman and Buildah
 Requires: %{name} = %{epoch}:%{version}-%{release}
 Requires: container-network-stack
 Requires: oci-runtime
+Requires: iptables
+Requires: nftables
+Requires: passt
+%if %{defined fedora}
 Conflicts: podman < 5:5.0.0~rc4-1
 Recommends: composefs
 Recommends: crun
```

Comment:
This should be global?

Reply:
The crun and netavark dependencies are already handled by the global:
```
Requires: oci-runtime
Requires: container-network-stack
```
so we should be fine with having crun and netavark inside Fedora, so people can use runc, cni if they want.

Reply:
@rhatdan for RHEL10 runc and cni is gone - it is no longer available there.